File: app/src/main/java/me/pwcong/radarmapviewdemo/MainActivity.java
package me.pwcong.radarmapviewdemo;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

import java.util.ArrayList;

import me.pwcong.radarmapview.model.RadarMapEntry;
import me.pwcong.radarmapview.view.RadarMapView;

public class MainActivity extends AppCompatActivity {

    RadarMapView radarMapView;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        radarMapView= (RadarMapView) findViewById(R.id.radarMapView);
        radarMapView.setData(getList());

    }

    private ArrayList<RadarMapEntry> getList(){

        ArrayList<RadarMapEntry> entries=new ArrayList<>();

        entries.add(new RadarMapEntry("小彭",26));
        entries.add(new RadarMapEntry("小花",15));
        entries.add(new RadarMapEntry("小黑",23));
        entries.add(new RadarMapEntry("小红",18));
        entries.add(new RadarMapEntry("小黄",33));
        entries.add(new RadarMapEntry("小可",43));

        return entries;

    }

}

File: app/src/main/java/me/pwcong/radarmapviewdemo/MainActivity.kt
package me.pwcong.radarmapviewdemo

import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity
import me.pwcong.radarmapview.entry.RadarMapEntry
import me.pwcong.radarmapview.view.RadarMapView

class MainActivity : AppCompatActivity() {
    var radarMapView: RadarMapView? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        radarMapView = findViewById<View>(R.id.radarMapView) as RadarMapView
        radarMapView!!.setData(list)
    }

    private val list: List<RadarMapEntry>
        get() {
            val entries = mutableListOf<RadarMapEntry>()

            entries.add(RadarMapEntry("小彭", 26F))
            entries.add(RadarMapEntry("小花", 15F))
            entries.add(RadarMapEntry("小黑", 23F))
            entries.add(RadarMapEntry("小红", 18F))
            entries.add(RadarMapEntry("小黄", 33F))
            entries.add(RadarMapEntry("小可", 43F))

            return entries
        }
}
